weeb/module.py

```diff
@@ -0,0 +1,143 @@
+import discord
+from discord.ext import commands
+from discord.abc import PrivateChannel
+import aiohttp
+from typing import Optional
+from core import utils, text, logging, i18n
+
+_ = i18n.Translator("modules/fun").translate
+
+
+class Weeb(commands.Cog):
+    """
+    Weeby commands for your own Weebiness
+
+    Includes:
+        ?sauce that will give your desired sauce if you supply it with a magic number
```

Comment:
I don't think this `Includes:` section is necessary. You can see the commands of a module using the `help <Cog>` command.

Reply:
+1